src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -1321,7 +1327,16 @@ public function getAllSchemas()
      */
     public function getColumnTypes()
     {
-        return array_merge(parent::getColumnTypes(), ['json', 'jsonb', 'cidr', 'inet', 'macaddr', 'interval']);
+        return array_merge(parent::getColumnTypes(), [
+            'json',
+            'jsonb',
+            'xml',
+            'oid',
+            'cidr',
+            'inet',
+            'macaddr',
+            'interval'
```

Comment:
```suggestion
            'interval',
```
Add comma for future diffs to be less noisy